api/src/organizational-entities/infrastructure/serializers/jsonapi/organizations-administration/organization-for-admin-serializer.js

```diff
@@ -1,7 +1,7 @@
 import { Serializer } from 'jsonapi-serializer';
 import _ from 'lodash';
 
-import { OrganizationForAdmin } from '../../../../../src/organizational-entities/domain/models/OrganizationForAdmin.js';
+import { OrganizationForAdmin } from '../../../../domain/models/OrganizationForAdmin.js';
```

Comment:
**Remarque** : Si on suit les conventions d'Accès, il faudrait un point entre admin et serializer => `organization-for-admin.serializer.js`

Reply:
C'est accès qui gèrent ce scope ?

Reply:
si c'est `organizational-entities` oui :)